### PR TITLE
Populate TestTable with multi-row INSERT

### PR DESCRIPTION
**Pull request by findepi**
Use single INSERT instead of multiple single-row inserts. Should be
    faster, especially e.g. for Trino.

### CODE REVIEW

**Review by findinpath — APPROVED** [review]
Please add in the description of the PR, for posterity, if you spot more dramatic test runtime improvements.

File: plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BigQueryTestView.java [hunk 1]
```diff
@@ -14,32 +14,24 @@
...
-        extends TestTable
+        implements TemporaryRelation
 {
-    private final TestTable table;
+    private final SqlExecutor sqlExecutor;
+    private final TemporaryRelation table;
     private final String viewName;
 
-    public BigQueryTestView(SqlExecutor sqlExecutor, TestTable table)
+    public BigQueryTestView(SqlExecutor sqlExecutor, TemporaryRelation table)
```
Comment (findinpath) [inline plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BigQueryTestView.java:29]: nit/optional: rename `table` to `relation`

File: plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BigQueryTestView.java [hunk 2]
```diff
@@ -51,7 +43,7 @@ public String getName()
     @Override
     public void close()
     {
-        sqlExecutor.execute("DROP TABLE " + table.getName());
+        table.close();
```
Comment (findinpath) [inline plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BigQueryTestView.java:46]: pre-existing: shouldn't we drop the view first and then the underlying relation?

**Review by hashhar — APPROVED** [review]

File: plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BigQueryTestView.java (hunk 1 above)
Comment (findepi) [reply plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BigQueryTestView.java:29]: thanks, renaming in the commit in which we no longer assume this is a table.

File: plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BigQueryTestView.java (hunk 2 above)
Comment (findepi) [reply plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BigQueryTestView.java:46]: will fix too

**Comment by hashhar** [comment]
thanks a lot Piotr. I already see minimum of 3m savings per JDBC connector test. It'll be more for slow connectors like Redshift (but we don't run those on CI today).

**Pull request merged**